### ISSUE TITLE
Progression :suggestion:

### ISSUE THREAD

**Issue by Kerodas** [naruto-unison/naruto-unison#43]
I suggest making a progression system for characters based on Naruto arcs. 180 characters is a huge roster, adding those characters over a period of time should serve better in a long term of game existence. And it should help with scaling characters. I suggest making available (both open and locked) characters till Leaf Destruction arc. After that expanding roster with new arc every month.

Also, we need a mission status window either in a separate page like in old n-a or when hovering cursor over locked character window.

**Comment by jnbooth** [comment]
I've been very torn about your first point, so I really appreciate your feedback! The counter-argument is that this is supposed to be an upgrade to Naruto Arena, so it's arguable that one of its key advantages over other attempts at resurrecting N-A is having all of N-A's characters from the get-go. I think you're probably right, though, especially so that it will be easier to balance characters a few at a time. The idea of adding by arc is very neat. Ultimately, I think this will have to come down to a collective vote during beta. In the meantime, all the characters will be visible simply to make testing easier.

Regarding your second point, both of those sound like good approaches, so which do you favor? I suppose it could be both. At present, missions are displayed on character pages (from the main site), so another option would be to add a link in locked character windows to their page. Anyway, I can definitely add a mission overview page. I never liked the N-A approach of hiding mission objectives under "You have not completed the prerequisites for this mission," so all of them would be visible regardless, but other than that I could make a fairly close replica.

By the way, don't worry about adding labels in the title like :suggestion:. GitHub's labeling system is sufficient.

**Comment by Kerodas** [comment]
I think that everything regarding the game should be displayed in the game to make user experience as comfortable as possible, I didn't like how n-a had info distributed in various sections of the website, having to have multiple tabs open, although I think both approaches for mission status are good to have. A website page is good for archive purposes while in-game is good for players.

**Comment by jnbooth** [comment]
How's this?

**Comment by RadoUzee** [comment]
I agree with the idea to release them over time. Even though it's a continuation of naruto-arena i still feel like it will result in a better expirience for the players. The game will be harder to properly balance if all are in the game from the start.
If you put everything out at once it'll result in the need of constant balance changes to keep up with the needs of players. Whereas if at start we only have the basic charracters + around 20 unlockable charracters(by basic i mean only the earlier chars, i think a lot more need missions than are already set to missions) we can see how meta develops, add chars that we think can shake up the meta when it gets too stale. And also make slight changes to existing chars overtime( weekly updates to keep the game fresh). 
PS: And i truly love the idea of showing missions inside the client
